docs/UserGuide.md

```diff
@@ -115,11 +115,118 @@ Examples:
 
-Format: `list`
+Format: `list client`
+
+### Adding a booking: `add booking`
+
+Adds a booking for a client to the application.
+
+Format: `add booking p/PHONE_NUMBER s/SLOT_NUMBER`
+
+<div markdown="block" class="alert alert-primary">
+
+**:bookmark: Note:**<br>
+
+* <code>p/PHONE_NUMBER</code> is the phone number of the client.
+
+* <code>s/SLOT_NUMBER</code> is the time slot number which the client needs to specify.<br>
+
+Exhaustive list of time slot numbers: 
+
+Time slot number | Time period
+:---------------:|:------------------:
+1                | 0000 hrs - 0130 hrs
+2                | 0130 hrs - 0300 hrs
+3                | 0300 hrs - 0430 hrs
+4                | 0430 hrs - 0600 hrs
+5                | 0600 hrs - 0730 hrs
+6                | 0730 hrs - 0900 hrs
+7                | 0900 hrs - 1030 hrs
+8                | 1030 hrs - 1200 hrs
+9                | 1200 hrs - 1330 hrs
+10               | 1330 hrs - 1500 hrs
+11               | 1500 hrs - 1630 hrs
+12               | 1630 hrs - 1800 hrs
+13               | 1800 hrs - 1930 hrs
+14               | 1930 hrs - 2100 hrs
+15               | 2100 hrs - 2230 hrs
+16               | 2230 hrs - 0000 hrs
+
+
+* Please refer to the examples below.
+
+</div>
+
+Examples:
+* `add booking p/89653101 s/7` Adds a booking at 0900hrs to 1030hrs for the client who has 89653101 as their phone number.
+
+### Deleting a booking: `delete booking`
+
+Deletes a booking from the application.
+
+Format: `delete booking INDEX`
+
+Examples:
+* `delete booking 1` Deletes the client at index 1 in the booking list currently shown.
+
+### Finding booking by keywords: `find booking`
+
+Finds booking whose attribute(s) matches the keyword(s).
+
+Format: `find booking [d/DATE] [t/START_TIME] [n/NAME] [s/STATUS]`
+
+<div markdown="block" class="alert alert-primary">
+
+**:bookmark: Note:**<br>
+
+* The search is case insensitive.
+
+* There must be 1 or more search arguments.
+
+* Date and time must follow the format specified [above](#features). 
+
+* Partial search will be allowed. <br>
+  e.g. <code>find booking n/Al</code> can show bookings for clients with names like Alice and Alex.
+
+* A booking either has no status (which represents a booking that is not completed) or a completed status.
+
+</div>
+
+Examples:
+* `find booking n/al` Find bookings for clients with names matching 'al'. E.g. Alex, Alice, Al.
+* `find booking d/12-12-2021 t/1930` Find all bookings on 12 December 2021 which starts at 1930 hrs.
+* `find booking s/c` Find all completed bookings.
+
+### Listing all bookings: `list booking`
+
+Lists all bookings in the application.
+
+Format: `list booking`
+
+### Mark booking as done: `done booking`
+
+Mark booking as done once the client has entered the gym on their time slot
+
+Format: `done booking d/DATE s/SLOT_NUMBER p/PHONE_NUMBER`
+
+<div markdown="block" class="alert alert-primary">
+
+**:bookmark: Note:**<br>
+
+* Date must follow the format specified [above](#features).
```

Comment:
Could you also add this point to the note for `add client` and `find client` just for consistency? Thank you